charts/pathfinder-rpc-server/values.yaml

```diff
@@ -0,0 +1,36 @@
+# LetsEncrypt certificates
+acmeEmail:                 circlesubi@gmail.com
+acmeServer:                https://acme-v02.api.letsencrypt.org/directory
+
+# Smart Contracts
+contracts:
+  hubAddress:
+
+
+
+
+# Pathfinder proxy
+pathfinderProxy:
+  port: 8675
+  upstreamServiceEndpoints: http://pathfinder2:54389
+  upstreamHealthEndpoints:  http://pathfinder2-updater:8794
+
+# Pathfinder updater
+pathfinderUpdater:
+  indexerDBConnectionString:
+  indexerWSurl:              ws://blockchain-indexer:8675
+  internalCapacityGraph:     /var/pathfinder2/data/capacity_graph.db
+  externalCapacityGraph:     /var/pathfinder2/data/capacity_graph.db
+  rpcURL:                    http://pathfinder2:54389
+
+postgresql:
+  enabled: true 
+  auth:
+    username:                 doadmin
+    database:                 index
+    existingSecret:           indexer-db-secret 
+  primary:
+    service:
+      type:                   NodePort
+      nodePorts:
+        postgresql:           "30032"
```

Comment:
unless there is another method to pg_restore the db  I am afraid we need this

Reply:
pg_restore should be available in the Bitnami Postgres containers.

Merging the above example with what you wrote in the README, we'd get:

```
kubectl -n pathfinder exec -it pod/pathfinder-postgresql-0 -- sh -c 'pg_restore -h localhost -p 5432 -U ${POSTGRES_USER} -v -d ${POSTGRES_DB}' < indexer.dump
```

This can be useful when dealing with natively compressed dumps.

Reply:
yes, makes sense. Updating docs

Reply:
I cannot find your other `pg_restore` comment which I was notified about, why I am responding here:

As we need to use `pg_restore` and cannot use `psql`, due to the compressed database dump created with `pg_dump -FcZ`, and with `pg_restore` missing a flag to provide the database password non-interactively, we have two options:

1. Supply a connection string for the `-d` database parameter. [Reference](https://www.postgresql.org/docs/current/app-pgrestore.html).  
  ```
  kubectl -n pathfinder exec -it pod/pathfinder-postgresql-0 -- sh -c 'pg_restore -v -d "postgres://${POSTGRES_USER}:${POSTGRES_PASSWORD}@localhost:5432/${POSTGRES_DB}"' < indexer.dump
  ```
2. Supply a respective `PGPASSWORD` via the environment:  
  ```
  kubectl -n pathfinder exec -it pod/pathfinder-postgresql-0 -- sh -c 'PGPASSWORD=${POSTGRES_PASSWORD} pg_restore -h localhost -p 5432 -U ${POSTGRES_USER} -v -d ${POSTGRES_DB}' < indexer.dump
  ```